exporters/otlp/src/main/java/io/opentelemetry/exporters/otlp/OtlpGrpcSpanExporter.java

```diff
@@ -196,6 +274,27 @@ protected Builder fromConfigMap(
       if (value != null) {
         this.setDeadlineMs(value);
       }
+      String endpointValue = getStringProperty(KEY_ENDPOINT, configMap);
+      if (endpointValue != null) {
+        this.setEndpoint(endpointValue);
+      }
+
+      Boolean useTlsValue = getBooleanProperty(KEY_USE_TLS, configMap);
+      if (useTlsValue != null) {
+        this.setUseTls(useTlsValue);
+      }
+
+      String metadataValue = getStringProperty(KEY_METADATA, configMap);
+      if (metadataValue != null) {
+        for (String keyValueString : Splitter.on(';').split(metadataValue)) {
+          final List<String> keyValue = Splitter.on('=').splitToList(keyValueString);
```

Comment:
Believe you can use `Splitter.withKeyValueSeparator`

Reply:
Would be good to create a static final with the splitter

`private static final Splitter.MapSplitter METADATA_PROPERTY_SPLITTER = Splitter.on(';').trimResults().omitEmptyStrings().withKeyValueSplitter('=');

Reply:
`withKeyValueSeparator ` is marked with `@Beta`, should we use it?

Reply:
Oof - good catch, I always forget that. Yeah let's not use it.